src/ZendeskApi_v2/Models/Shared/Event.cs

```diff
@@ -12,7 +12,7 @@ public class Event
         public string Type { get; set; }
 
         [JsonProperty("body")]
-        public string Body { get; set; }
+        public object Body { get; set; }
```

Comment:
```suggestion
        public dynamic Body { get; set; }
```

I think using a dynamic will make accessing nested data easier 
for more info:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/using-type-dynamic